deployments/properties/octopusdeploy.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Build and Deploy with Octopus Deploy",
+    "description": "Build a docker container, create a release in Octopus Deploy and deploy it to your environment.",
+    "creator": "Octopus Deploy",
+    "iconName": "octopus-deploy",
```

Comment:
unclear what `iconName` should be set to.
The name is `octopus-deploy` in font-awesome but I'm not expecting that is what is being used here as they look to be higher quality